interface/main/left_nav.php

```diff
@@ -160,9 +160,6 @@
   'dld' => array(xl('Display Documents'), 0, 'main/display_documents.php')
  );
  $primary_docs['npa']=array(xl('Batch Payments')   , 0, 'billing/new_payment.php');
- if ($GLOBALS['use_charges_panel'] || $GLOBALS['concurrent_layout'] == 2) {
-  $primary_docs['cod'] = array(xl('Charges'), 2, 'patient_file/encounter/encounter_bottom.php');
- }
```

Comment:
Add a menu_styling_vertical global which does:
0 for Tree
1 for Sliding
(and incorportate if this global == 0 above in above if)